themes/bootstrap3/templates/search/home.phtml

```diff
@@ -15,6 +15,10 @@
 ?>
 
 <div class="searchHomeContent">
+  <?=$this->component('confirm-menu', [
+    'label' => 'Delete everything',
```

Comment:
Is this a test instance that has been committed by accident? Should we instead incorporate the component by refactoring real existing code?